starknet_devnet/blueprints/rpc/structures/responses.py

```diff
@@ -85,6 +85,14 @@ class RpcEventsResult(TypedDict):
     continuation_token: str
 
 
+class RpcEventsResultWithoutContinuationToken(TypedDict):
```

Comment:
Can't we use class `RpcEventsResult` with continuation token being `Optional` (imported from `typing`)

Reply:
I tried something like this: 
```
    if events:
        continuation_token = continuation_token + 1
        continuation_token = str(continuation_token)
    else:
        continuation_token = None

    return RpcEventsResult(events=events, continuation_token=continuation_token)
```
and
```
class RpcEventsResult(TypedDict):
    """
    TypedDict for rpc get events result
    """

    events: List[EmittedEvent]
    continuation_token: Optional[str]
```
but then when I called an event 4 times, then called `get_events` with `"chunk_size": 1` and followed continuation token, when called with `"continuation_token": "4"` devnet returned INTERNAL_ERROR code (-32603)
![image](https://user-images.githubusercontent.com/23462504/229752208-079d8d0d-dd7b-450a-9e00-297f90ea55e4.png)